src/sodiumCompatibility/java/net/coderbot/iris/compat/sodium/impl/options/IrisSodiumOptions.java

```diff
@@ -21,7 +21,7 @@ public static OptionImpl<Options, Integer> createMaxShadowDistanceSlider(Minecra
         OptionImpl<Options, Integer> maxShadowDistanceSlider = OptionImpl.createBuilder(int.class, vanillaOpts)
                 .setName(Component.translatable("options.iris.shadowDistance"))
                 .setTooltip(Component.translatable("options.iris.shadowDistance.sodium_tooltip"))
-                .setControl(option -> new SliderControl(option, 0, 32, 1, ControlValueFormatter.quantityOrDisabled("Chunks", "Disabled")))
+                .setControl(option -> new SliderControl(option, 0, 32, 1, ControlValueFormatter.quantityOrDisabled(Text.translatable("options.chunks"), Text.translatable("options.off"))))
```

Comment:
The chunks probably don't get the value right now, as `options.chunks` is "%s chunks" and therefore needs an inserted value to the variable.